commands/command.go

```diff
@@ -30,20 +30,21 @@ var Verbs = map[string]func(api.API) Command{
 	"paste":     NewPaste,
 	"pause":     NewPause,
 	"play":      NewPlay,
-	"prev":      NewPrevious,
 	"previous":  NewPrevious,
+	"prev":      NewPrevious,
```

Comment:
Seems it's ignoring the spaces (and possibly the `":`), and sorting `i` before `N`. Same on yours or not?